Dockerfile

```diff
@@ -16,10 +16,13 @@ RUN  mkdir /vcs_scanner
 
 COPY ./ /vcs_scanner
 
+RUN wget https://github.com/gitleaks/gitleaks/releases/download/v8.18.4/gitleaks_8.18.4_linux_arm64.tar.gz \
+&& tar -zxvf gitleaks_8.18.4_linux_arm64.tar.gz -C /vcs_scanner/gitleaks_config/
```

Comment:
did you check the name of the executable match? :)

Reply:
Yes.